test/system/010-images.bats

```diff
@@ -410,19 +424,19 @@ EOF
 FROM $IMAGE
 RUN echo $i
 EOF
-        run_podman build -q -t i$i $PODMAN_TMPDIR
+        run_podman build -q -t img-$i-$(safename) $PODMAN_TMPDIR
     done
 
     run_podman images
     # Now remove all images in parallel and in the background and make sure
     # that listing all images does not fail (see BZ 2216700).
     for i in $(seq --format '%02g' 1 $count); do
         timeout --foreground -v --kill=10 60 \
-                $PODMAN rmi i$i &
+                $PODMAN rmi img-$i-$(safename) &
     done
 
     tries=100
-    while [[ ${#lines[*]} -gt 1 ]] && [[ $tries -gt 0 ]]; do
+    while [[ "$output" =~ "$(safename)" ]] && [[ $tries -gt 0 ]]; do
         # Prior to #18980, 'podman images' during rmi could fail with 'image not known'
         # '0+w' because we sometimes get warnings.
         run_podman 0+w images --format "{{.ID}} {{.Names}}"
```

Comment:
wait the warning that we ignore here here was fixed https://github.com/containers/podman/issues/23331 so this can/should be removed